data/properties.json

```diff
@@ -0,0 +1,509 @@
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 27,
+	"name": "is translator of",
+	"name_reverse": "has translator",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 28,
+	"name": "is editor of",
+	"name_reverse": "has editor",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 29,
+	"name": "is director of",
+	"name_reverse": "has director",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 30,
+	"name": "is dramaturg for",
+	"name_reverse": "has dramaturg",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 31,
+	"name": "is composer of",
+	"name_reverse": "has composer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 32,
+	"name": "is session musician on",
+	"name_reverse": "has session musician",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 33,
+	"name": "is stage designer for",
+	"name_reverse": "has stage designer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 34,
+	"name": "is costume designer for",
+	"name_reverse": "has costume designer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 35,
+	"name": "is make-up artist on",
+	"name_reverse": "has make-up artist",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 36,
+	"name": "is sound engineer of",
+	"name_reverse": "has sound engineer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 37,
+	"name": "is film editor of",
+	"name_reverse": "has film editor",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 38,
+	"name": "is actor in",
+	"name_reverse": "has actor",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 39,
+	"name": "is narrator of",
+	"name_reverse": "has narrator",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 40,
+	"name": "is cinematographer of",
+	"name_reverse": "has cinematographer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 41,
+	"name": "is head of production of",
+	"name_reverse": "has head of production",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+},
+{
+	"id": 42,
+	"name": "is production designer of",
+	"name_reverse": "has production designer",
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+}
+]
```

Comment:
This file should also end with a trailing newline.

Reply:
also updated